backend/src/models/events.py

```
@@ -157,7 +162,9 @@ def validate_dates(cls, v):
                     parsed_date = datetime.strptime(v, '%d %B %Y')
                     return parsed_date.strftime('%Y-%m-%d')
                 except ValueError:
-                    raise ValueError(f"Invalid date format: {v}. Expected YYYY-MM-DD or DD MMM YYYY")
+                    raise ValueError(
+                        f"Invalid date format: {v}. Expected YYYY-MM-DD or DD MMM YYYY"
+                    )
```

Ruff (0.13.1)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)